setup.py

```diff
@@ -405,7 +405,7 @@ def _env_variable_bool(key, default):
     long_description = "\n".join(long_description_s[long_description_s.index("INTRODUCTION"):])
 
 setup (name = 'phoebe',
-       version = '2.4.4',
+       version = '2.4.5',
        description = 'PHOEBE devel version',
```

Comment:
while we're at it, can you update this?

```suggestion
       description = 'PHOEBE 2.4',
```